docs/apis/batch/libs/gelly.md

```diff
@@ -1734,3 +1734,547 @@ vertex represents a group of vertices and each edge represents a group of edges
 vertex and edge in the output graph stores the common group value and the number of represented elements.
 
 {% top %}
+
+Graph Generators
+-----------
+
+Gelly provides a collection of scalable graph generators. Each generator is
+
+* parallelizable, in order to create large datasets
+* scale-free, generating the same graph regardless of parallelism
+* thrifty, using as few operators as possible
+
```

Comment:
Could we add an overview of how graph generators can be created and used? e.g. that you pass the parameters to the specific generator and then call `generate()` to get the graph?